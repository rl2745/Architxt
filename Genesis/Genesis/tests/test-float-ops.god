void init(){}
void update(int f){}

int main()
{
  float a;
  float b;
  float c;
  float d;
  float e;
  float f;
  int g;
  float h;

  a = 5.1;
  b = 4.7;
  c = a + b;
  d = a - b;
  e = a + a / d;
  f = a * 3;
  g = 3;
  h = a * g;

  printfl(a);
  printfl(b);
  printfl(c);
  printfl(d);
  printfl(e);
  printfl(f);
  printfl(h);
  return 0;
}
