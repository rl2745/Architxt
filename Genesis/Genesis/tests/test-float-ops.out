5.100000
4.700000
9.800000
0.400000
17.850000
15.300000
15.300000
